core/payment-driver/zksync/src/dao.rs

```diff
@@ -56,6 +62,8 @@ impl ZksyncDao {
         let recipient = msg.recipient().to_owned();
         let gnt_amount = utils::big_dec_to_u256(msg.amount());
         let gas_amount = Default::default();
+        // TODO: Remove unwrap? default to rinkeby?
+        let (network, _token) = platform_to_network_token(msg.platform()).unwrap();
```

Comment:
Error should be propagated to `schedule_payment`. Invalid platform would be a sign that there's a bug in payment service,

Reply:
good one, updated return value and removed unwrap